tests/e2e/lib/virt_helpers.go

```diff
@@ -0,0 +1,408 @@
+	}
+	if !ok {
+		return "", nil, errors.New("failed to get current CSV from " + name + " packagemanifest")
+	}
+	log.Printf("Current CSV is: %s", csv)
+
+	versionString, ok, err := unstructured.NestedString(firstChannel, "currentCSVDesc", "version")
+	if err != nil {
+		return "", nil, err
+	}
+	if !ok {
+		return "", nil, errors.New("failed to get current operator version from " + name + " packagemanifest")
+	}
+	log.Printf("Current operator version is: %s", versionString)
+
+	version, err := version.ParseGeneric(versionString)
+	if err != nil {
+		return "", nil, err
+	}
+
+	return csv, version, nil
+}
+
+// Checks the existence of the operator's target namespace
+func (v *VirtOperator) checkNamespace() bool {
+	// First check that the namespace exists
+	exists, _ := DoesNamespaceExist(v.Clientset, v.Namespace)
+	return exists
+}
+
+// Checks for the existence of the virtualization operator group
+func (v *VirtOperator) checkOperatorGroup() bool {
+	group := operatorsv1.OperatorGroup{}
+	err := v.Client.Get(context.TODO(), client.ObjectKey{Namespace: v.Namespace, Name: "kubevirt-hyperconverged-group"}, &group)
+	if err != nil {
+		return false
+	}
+	return true
+}
+
+// Checks if there is a virtualization subscription
+func (v *VirtOperator) checkSubscription() bool {
+	subscription := operatorsv1alpha1.Subscription{}
+	err := v.Client.Get(context.TODO(), client.ObjectKey{Namespace: v.Namespace, Name: "hco-operatorhub"}, &subscription)
+	if err != nil {
+		return false
+	}
+	return true
+}
+
+// Checks if the ClusterServiceVersion status has changed to ready
+func (v *VirtOperator) checkCsv() bool {
+	subscription, err := v.GetOperatorSubscription()
+	if err != nil {
+		if err.Error() == "no subscription found" {
+			return false
+		}
+	}
+
+	return subscription.CsvIsReady(v.Client)
+}
+
+// CheckHco looks for a HyperConvergedOperator and returns whether or not its
+// health status field is "healthy". Uses dynamic client to avoid uprooting lots
+// of package dependencies, which should probably be fixed later.
+func (v *VirtOperator) checkHco() bool {
+	resourceId := schema.GroupVersionResource{
+		Group:    "hco.kubevirt.io",
+		Resource: "hyperconvergeds",
+		Version:  "v1beta1",
+	}
+
+	unstructuredHco, err := v.Dynamic.Resource(resourceId).Namespace(v.Namespace).Get(context.Background(), "kubevirt-hyperconverged", v1.GetOptions{})
+	if err != nil {
+		log.Printf("Error getting HCO: %v", err)
+		return false
+	}
+
+	health, ok, err := unstructured.NestedString(unstructuredHco.UnstructuredContent(), "status", "systemHealthStatus")
+	if err != nil {
+		log.Printf("Error getting HCO health: %v", err)
+		return false
+	}
+	if !ok {
+		log.Printf("HCO health field not populated yet")
+		return false
+	}
+	log.Printf("HCO health status is: %s", health)
+
+	return health == "healthy"
+}
+
+// Creates the target virtualization namespace, likely openshift-cnv or kubevirt-hyperconverged
+func (v *VirtOperator) installNamespace() error {
```

Comment:
I would not do this kind of refactor in this PR. That can create questions as why not to use `tests/e2e/lib/kube_helpers.go` for this? It can be follow up PR

Reply:
When common_k8s_utils is merged, installNamespace would be a good candidate to put there. But probably named CreateNamespace or something. Maybe installOperatorGroup too, but probably not the others.